src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.stories.tsx

```diff
@@ -19,93 +19,68 @@ const meta: Meta<typeof DashboardEl> = {
 export default meta;
 type Story = StoryObj<typeof DashboardEl>;
 
-const BreachMockItem1: HibpLikeDbBreach = {
-  AddedDate: new Date("2018-11-07T14:48:00.000Z"),
-  BreachDate: "11/09/23",
-  DataClasses: ["email-addresses", "passwords"],
-  Description: "",
-  Domain: "",
-  Id: 0,
-  IsFabricated: false,
-  IsResolved: true,
-  IsMalware: false,
-  IsRetired: false,
-  IsSensitive: false,
-  IsSpamList: false,
-  IsVerified: false,
-  LogoPath: "",
-  ModifiedDate: new Date("2011-12-15T14:48:00.000Z"),
-  Name: "",
-  PwnCount: 0,
-  Title: "Twitter",
+const BreachMockItem1: SubscriberBreach = {
+  addedDate: "2023-06-18T14:48:00.000Z",
+  breachDate: "11/09/23",
+  dataClasses: ["email-addresses", "ip-addresses", "phone-numbers"],
+  description: "",
+  domain: "",
+  id: 3,
+  modifiedDate: "2013-12-07T14:48:00.000Z",
+  name: "",
+  title: "Facebook",
+  favIconUrl: "",
+  emailsEffected: ["email1@gmail.com", "email2@yahoo.com"],
```

Comment:
Q: should we create some generator to create mock results (similar to `createRandomScan()` below)?
Or do we need them to have specific `dataClasses` and/or `emailsEffected`?

```js
const scannedResultsArraySample: ScanResult[] = Array.from(
  { length: 5 },
  createRandomScan
);
```